backend/src/migrations/versions/128a654019d2_initial.py

```diff
@@ -5,46 +5,97 @@
 Create Date: 2021-11-04 20:55:07.442666
 
 """
-from alembic import op
+import uuid
+
 import sqlalchemy as sa
+from alembic import op
 from sqlalchemy.dialects import postgresql
 
 # revision identifiers, used by Alembic.
-revision = '128a654019d2'
+revision = "128a654019d2"
 down_revision = None
 branch_labels = None
 depends_on = None
 
 
 def upgrade():
-    # ### commands auto generated by Alembic - please adjust! ###
-    op.create_table('users',
-    sa.Column('id', postgresql.UUID(as_uuid=True), nullable=False),
-    sa.Column('created_at', sa.TIMESTAMP(timezone=True), nullable=False),
-    sa.Column('name', sa.String(), nullable=True),
-    sa.Column('password', sa.String(), nullable=True),
-    sa.PrimaryKeyConstraint('id')
+    # Mostly auto-generated tables with alembic revision --autogenerate.
+    # Some manual tweaks for default values inherited from BaseDAO
+    # that didn't seem to get picked up by autogenerate
```

Comment:
Let's add both of these links into the README and maybe here as well.

- https://alembic.sqlalchemy.org/en/latest/autogenerate.html#auto-generating-migrations
- https://alembic.sqlalchemy.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect

Reply:
I've added both of those links to the docs.  I've also changed the actual revision to not make tweaks to the `op.create_table` commands, but rather set the `id` and `create_at` values in the seed data creation piece of this.  

When there are actual gotchas in table schemas with `alembic revision --autogenerate`, we should definitely point that out.  There's not really a gotcha here, so much as a the fact that for seed data creation we're doing it by hand instead of using the SQLAlchemy models in models.py.